cmake/ActsCodegen.cmake

```diff
@@ -84,7 +84,12 @@ function(acts_code_generation)
     )
 
     if(NOT DEFINED ARGS_PYTHON_VERSION)
-        set(ARGS_PYTHON_VERSION "3.13")
+        if(ARGS_OFFLINE)
+            find_package(Python REQUIRED COMPONENTS Interpreter)
+            set(ARGS_PYTHON_VERSION ${Python_EXECUTABLE})
```

Comment:
At the very least this should lock down the exact python version (3.13 right now) to ensure compatibility. 

The goal was to avoid any kind of breakage due to user setups and this was part of that.